Gemfile

```diff
@@ -2,7 +2,7 @@
 
 git_source(:github) { |name| "https://github.com/#{name}.git" }
 source "https://rubygems.org"
-ruby File.read(".ruby-version").strip
+ruby File.read(File.join(File.dirname(__FILE__), ".ruby-version")).strip
```

Comment:
This is the way I did it in my former companies [template](https://github.com/lockstep/rails_new), so pretty much the same:

```ruby
# Only define Ruby version once (i.e. for Heroku)
version_file = File.join(File.dirname(__FILE__), '.ruby-version')
ruby File.read(version_file).strip
```